### PR TITLE
fix+feat: add --notification= option, fix zsh complain syntax

### PR DESCRIPTION
**Pull request by TChanhTinh**
- Add enable_noti=<1:default, 2> config that enable notification feature
- Add flag -N --notification <0,1> enable/disable notification feature
- Add notification flag -h entry
- If enable notification & thumbnails it will use thumbnail as notification icon else it will use default thumbnail icon (user defined) file at ./config/ytfzf/default_thumb.png
- Fix zsh complain syntax at download_thumbnails

### CODE REVIEW

File: ytfzf [hunk 1]
```diff
@@ -151,6 +153,7 @@ basic_helpinfo () {
     printf "     -l, --loop           <search-query>    Loop: prompt selector again after video ends\n";
     printf "     -s, --search-again   <search-query>    After the video ends make another search \n";
     printf "     -L, --link-only      <search-query>    Prints the selected URL only, helpful for scripting\n";
+    printf "     -N, --notification   <0,1>  			Send notification when play video\n";
```
Comment (pystardust) [inline ytfzf:156]: It doesn't need a 0 or a 1 right?
We could just leave that part empty

Comment (pystardust) [reply ytfzf:156]: if you are making it in such a way that it takes it arguemnt, then use --notification=
and also for parse opt you would need to send the argument as `"$2"`, refer the other flags that are already been done.

File: ytfzf [hunk 2]
```diff
@@ -995,13 +1018,15 @@ parse_opt () {
...
+		N)	YTFZF_NOTI=${optarg:-1} ;;
+
 		*)
 			usageinfo
 			exit 2 ;;
 	esac
 }
 
-while getopts "LhDmdfxHaArltSsvn:U:-:" OPT; do
+while getopts "LhDmdfxHaArltSsvnN:U:-:" OPT; do
```
Comment (pystardust) [inline ytfzf:1029]: here option `n` also takes a `OPTARG`, it should be `n:`

Comment (TChanhTinh) [reply ytfzf]: Thanks for the tips, was a little confuse at that, now flag -N now is work without arg i also change the -h description for it

File: ytfzf (hunk 1 above)
Comment (TChanhTinh) [reply ytfzf]: fixed on newer commit

**Comment by TChanhTinh** [comment]
Sorry for the mess, this is the first time is make a pull request, all problem you mentioned are solved.

**Comment by pystardust** [comment]
np, there are few more things we could do.
Here when we select multiple videos we get a error, we would need to adjust it so that even with multiselect we get a  formatted notification.

**Comment by TChanhTinh** [comment]
> np, there are few more things we could do.
> Here when we select multiple videos we get a error, we would need to adjust it so that even with multiselect we get a formatted notification.
Done in new commit, if user select more than one video, it will send notify "Add x video to play queue" & use default_thumb instead individual thumbnail

**Pull request merged**